chainercv/links/connection/convolution_2d_block.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+
+import chainer
+from chainer.functions import relu
+from chainer.links import BatchNormalization
+from chainer.links import Convolution2D
+
+
+class Convolution2DBlock(chainer.Chain):
+    """Convolution2D --> (Batch Normalization) --> Activation
```

Comment:
I agree that it is lengthy. 
However, I think that it is too dangerous to assume that all inputs are 2D.

Also, I realized that we are making an assumption that the activation function always comes last.
How about `Conv2DActivation` and `Conv2DBNActivation`?
So, `Bn->ReLU->Conv` would be `BNActivationConv2D`.